blockchain/types/tx_internal_data_ethereum_blob.go

```diff
@@ -221,6 +221,45 @@ func (sc *BlobTxSidecar) Copy() *BlobTxSidecar {
 	}
 }
 
+// ValidateWithBlobTx validates the blob transaction with the sidecar.
+// This method also verifies the validity of the proof.
+func (sc *BlobTxSidecar) ValidateWithBlobTx(tx *TxInternalDataEthereumBlob) error {
```

Comment:
How's about receiving blob hashes?
```suggestion
func (sc *BlobTxSidecar) ValidateWithBlobHashes(hashes []common.Hash) error {
```

Reply:
That's absolutely correct. Let me fix it!